hack/resolve-ip/main.go

```diff
@@ -0,0 +1,26 @@
+package main
+
+import (
+	"fmt"
+	"net"
+	"os"
+)
+
+func main() {
+
+	if len(os.Args) < 2 {
+		fmt.Fprintln(os.Stderr, "host argument missing")
+		os.Exit(1)
+	}
+	host := os.Args[1]
+	addrs, err := net.LookupHost(host)
+	if err != nil {
+		fmt.Fprintf(os.Stderr, "error: %+v\n", err)
```

Comment:
```suggestion
		fmt.Fprintf(os.Stderr, "error: %w\n", err)
```

related: https://github.com/redhat-et/apex/pull/389